### PR TITLE
Writers

### PR DESCRIPTION
**Pull request by AdalbertMemSQL**
Added Avro and BatchInsert Writers

### CODE REVIEW

File: src/main/scala/com/memsql/spark/MemsqlLoadDataWriter.scala [hunk 1]
```diff
@@ -156,12 +156,12 @@ class LoadDataWriterFactory(table: TableIdentifier, conf: MemsqlOptions)
     if (loadDataFormat == MemsqlOptions.LoadDataFormat.Avro) {
       new AvroDataWriter(avroSchema, outputstream, writer, conn)
     } else {
-      new LoadDataWriter(outputstream, writer, conn)
+      new CSVDataWriter(outputstream, writer, conn)
```
Comment (blinov-ivan) [inline src/main/scala/com/memsql/spark/MemsqlLoadDataWriter.scala:159]: why did you rename this writer

Comment (AdalbertMemSQL) [reply src/main/scala/com/memsql/spark/MemsqlLoadDataWriter.scala:159]: I think this name has more sense (we had AvroDataWriter and LoadDataWriter (which writes the CSV format))

File: src/main/scala/com/memsql/spark/v2/MemsqlWriteBuilder.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                       attemptNumber: Int,
+                       isReferenceTable: Boolean,
+                       mode: SaveMode,
+                       table: TableIdentifier,
+                       conf: MemsqlOptions)
+    extends BatchWrite
+    with LazyLogging {
+
+  override def createBatchWriterFactory(info: PhysicalWriteInfo): DataWriterFactory = {
+    log.debug(conf.toString + "!!!!!!!!!!!!!!")
```
[the remark this answers is not in the document]
Comment (AdalbertMemSQL) [reply src/main/scala/com/memsql/spark/v2/MemsqlWriteBuilder.scala]: removed

**Review by blinov-ivan — APPROVED** [review]
lgtm
